themes/godotengine/layouts/home.htm

```diff
@@ -241,48 +270,57 @@ <h4>Gorgeous 3D</h4>
           alt=""
           width="1"
           height="1"
-          style="background: linear-gradient(90deg, #252928 4%, #2c312f 13%, #252928 21%, #141213 33%)"
+          style="background: linear-gradient(90deg, #196f36 7%, #28674e 29%, #2a4b46 65%, #3b6f4e 97%)"
           loading="lazy"
         >
-        <div class="dark base-padding">
-          <h4>Beautiful 2D</h4>
-          <p>Dedicated 2D engine that works in pixel coordinates, with plenty of built-in tools.</p>
+        <div class="body dark">
+          <h4>True native 2D</h4>
+          <p>
+            Make crisp and performant 2D games with Godot's dedicated 2D
+            rendering engine with real 2D pixel coordinates and 2D nodes.
+          </p>
         </div>
       </div>
     </a>
 
-    <a href="/features#script" data-barba-prevent>
+    <a href="/features#features_3d" data-barba-prevent>
       <div class="feature">
         <img
-          src="{{ 'assets/home/features/easy_code.png' | theme }}"
+          src="{{ 'assets/home/features/3d.jpg' | theme }}"
           alt=""
           width="1"
           height="1"
-          style="background: linear-gradient(90deg, #252a35 46%, #252a35 53%, #202630 76%, #202630 89%)"
+          style="background: linear-gradient(90deg, #196f36 7%, #28674e 29%, #2a4b46 65%, #3b6f4e 97%)"
           loading="lazy"
         >
-        <div class="dark base-padding">
-          <h4>Easy to program</h4>
-          <p>Object-oriented API with language options such as GDScript, C#, C++ and visual scripting.</p>
+        <div class="body dark">
+          <h4>Simple and powerful 3D</h4>
+          <p>
+            Godot's 3D nodes give you everything you need to build, animate,
+            and render your 3D worlds and characters.
+          </p>
         </div>
       </div>
     </a>
   </div>
 
   <div class="flex eqsize responsive features-row">
-    <a href="/features#collaborate" data-barba-prevent>
+    <a href="https://docs.godotengine.org/en/stable/tutorials/export/exporting_basics.html#exporting-by-platform" data-barba-prevent>
```

Comment:
I would still link to a section on the Features page (that we need to create at the moment), so people don't leave the main website by accident. That section, in turn, can link to the documentation.

Reply:
Didn't realize there was a section for deployments

Reply:
Ah, right, me neither, I thought we need to create it still :D Even better, can totally be `#deploy`!